resource-aggregate/pb/commands.proto

```diff
@@ -385,3 +385,96 @@ message ConfirmResourceCreateResponse {
     AuditContext audit_context = 100;
 }
 
+// Update Device Metadata - Connection status                           
+                                                                                                    
+// ┌──────────┐          ┌───────┐                  ┌──────────────────┐               ┌─────────┐
+// │OCF Server│          │Gateway│                  │Resource Aggregate│               │Event Bus│
+// └──────────┘          └───┬───┘                  └────────┬─────────┘               └─────────┘
+//     ┌┴┐  SignInRequest   ┌┴┐                              ┌┴┐                            │     
+//     │ │ ───────────────> │ │                              │ │                            │     
+//     │ │                  │ │                              │ │                            │     
+//     │ │                  │ │ UpdateDeviceMetadataRequest  │ │                            │     
+//     │ │                  │ │ ────────────────────────────>│ │                            │     
+//     │ │                  │ │                              │ │                            │     
+//     │ │                  │ │ UpdateDeviceMetadataResponse │ │                            │     
+//     │ │                  │ │ <────────────────────────────│ │                            │     
+//     │ │                  └┬┘                              │ │                            │     
+//     │ │  SignInResponse   │                               │ │                            │     
+//     │ │ <─────────────────│                               │ │                            │     
+//     └┬┘                   │                               └┬┘                            │     
+//      │                    │                               │ DeviceMetadataUpdatePending  │     
```

Comment:
In this case it's DeviceMetadataUpdated as no confirmation is needed.

Reply:
fixed